src/js/script-loader.js

```diff
@@ -639,6 +639,24 @@ var LoaderProtoMethods = {
     },
 
     /**
+     * Reports a mismatched anonymous module error. Depending on the value of the configuration property
+     * `__CONFIG__.reportMismatchedAnonymousModules`, this method will throw an error, use the console[level]
+     * method to log the message or silently ignore it.
+     *
+     * @param  {string} msg Additional information to log with the error.
+     */
+    _reportMismatchedAnonymousModules: function(msg) {
+        var errorMessage = 'Mismatched anonymous define() module: ' + msg;
+        var reportLevel = this._config.reportMismatchedAnonymousModules;
+
+        if (!reportLevel || reportLevel === 'exception') {
+            throw new Error(errorMessage);
+        } else if (console && console[reportLevel]) {
+            console[reportLevel].call(console, errorMessage);
```

Comment:
Probably should've commented this out... we're using [stripDebug](https://github.com/sindresorhus/strip-debug) to build the production files, so `console[reportLevel]` gets stripped. This is the easiest way I found to overcome that.